src/components/Scorecards/ScorecardDetailsPage/CopyCsvButton.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2023 Cortex Applications, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { useCallback, useState } from 'react';
+import { CopyButton } from '../../Common/CopyButton';
+import FileCopyOutlinedIcon from '@material-ui/icons/FileCopy';
+import { ScorecardServiceScore } from '../../../api/types';
+import { humanizeAnyEntityRef } from '../../../utils/types';
+import { defaultComponentRefContext } from '../../../utils/ComponentUtils';
+import { percentify } from '../../../utils/NumberUtils';
+import { toCSV } from '../../../utils/collections';
+import { StringIndexable } from '../../ReportsPage/HeatmapPage/HeatmapUtils';
+import { HomepageEntity } from '../../../api/userInsightTypes';
+import { IconButton, Snackbar } from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
+
+interface CopyCsvButton {
+  entitiesByTag: StringIndexable<HomepageEntity>;
+  scores: ScorecardServiceScore[];
+}
+
+// eslint-disable-next-line @typescript-eslint/no-redeclare
+const CopyCsvButton = ({ entitiesByTag, scores }: CopyCsvButton) => {
```

Comment:
And then you can get rid of the eslint disable
```suggestion
const CopyCsvButton = ({ entitiesByTag, scores }: CopyCsvButtonProps) => {
```

Reply:
yup